lib/datatrans/config.rb

```diff
@@ -67,5 +77,13 @@ def web_transaction(*args)
     def xml_transaction(*args)
       XML::Transaction.new(self, *args)
     end
+
+    def json_transaction(*args)
+      JSON::Transaction.new(self, *args)
+    end
+
+    def start_json_transaction(*args)
```

Comment:
The other *_transaction() calls don't have an equivalent for this. Do we need it? Couldn't the caller just call `.transaction_path` on the `JSON::Transaction` themselves?

Reply:
Good point, will change it